esp/public/media/scripts/program/modules/scheduling_checks.jsx

```diff
@@ -114,3 +187,89 @@ var RefreshButton = React.createClass({
     </button>;
   },
 });
+
+/**
+ * Calls its onClick prop to reset table greying/sorting
+ */
+var ResetButton = React.createClass({
+  propTypes: {
+    onClick: React.PropTypes.func.isRequired,
+  },
+
+  render: function () {
+    return <button onClick={this.props.onClick} className="reset-button">
+      Reset
+    </button>;
+  },
+});
+
+// Modified from react-json-table example code.
+var SelectTable = React.createClass({
+  
+  propTypes: {
+    rows: React.PropTypes.array.isRequired,
+    saveState: React.PropTypes.shape({
+      greyed: React.PropTypes.object.isRequired,
+      sort: React.PropTypes.any.isRequired,
+      reverse: React.PropTypes.bool.isRequired
+    }).isRequired,
+    header: React.PropTypes.bool.isRequired,
+    columns: React.PropTypes.array,
+    clickHeader: React.PropTypes.func.isRequired,
+    clickRow: React.PropTypes.func.isRequired
+  },
+    
+  getInitialState: function(){
+    return {sort: this.props.saveState.sort, greyed: this.props.saveState.greyed, reverse: this.props.saveState.reverse};
```

Comment:
This state is no longer getting modified, so you may as well just remove it, and change the references to get everything directly from props.